documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,180 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+:toc:
+
+In the context of application development, Maven provides a standardized build system for projects. One of the main benefits of using Maven with your project is that it facilitates fetching dependencies from one or more repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
+
+. <<New_Project,Create a New Maven Project>>
+. <<New_Module,Create a New Maven Module>>
+. <<Maven_Support,Add Maven Support to an Existing Non-Maven Project>>
+
+Root Maven projects can serve as aggregators for multiple Maven modules, also known as sub-projects. For each module that is part of a maven project, a <module> entry is added to the project's pom.xml file. A pom.xml that contains <module> entries are often referred to as an aggregator pom.
+
+When modules are included into a project it is possible to execute Maven goals across all of the modules by a single command issued from the parent project directory.
+
+[[New_Project]]
+== 1. Create a New Maven Project
+The first step when using Maven with an Eclipse-based IDE is to create a Maven project. Maven projects may contain a single Maven module, multiple independent Maven modules or no Maven modules. Follow the instructions to create a Maven project:
```

Comment:
Maven projects can be standalone, or comprised of multiple sub-projects, a.k.a modules